### PR TITLE
feat: allow approximation of 1 kg in net-weight-verification [CARROT-1593]

### PR DESCRIPTION
**Pull request by GLGuilherme**
### Summary

_Allow approximation of 1 kg in net-weight-verification._

### Related links

_https://app.clickup.com/t/3005225/CARROT-1593_

---

- [X] **I, the PR author, declare that this PR works as expected and does not break any service. I also declare that I gave my best to apply all of the best practices and that I'm leaving the code better than I found it.**

## Summary by CodeRabbit

- **New Features**
	- Introduced a tolerance for net weight validation, allowing for approximate weight comparisons.
	- Added new test cases to enhance coverage for edge cases related to weight approximations.

- **Bug Fixes**
	- Updated output messages to clarify the new tolerance in weight validation.

- **Refactor**
	- Replaced static constants with dynamic functions for generating event attributes, improving adaptability to varying weight conditions.

### CODE REVIEW

**Review by gabrielsl96 — APPROVED** [review]

File: apps/methodologies/bold/rule-processors/mass/net-weight-verification/src/lib/net-weight-verification.processor.ts [hunk 1]
```diff
@@ -69,7 +70,7 @@ export class NetWeightVerificationProcessor extends RuleDataProcessor {
       eventIndex: number;
       loadNetWeight: number;
     } & CalculationValues) =>
-      `${this.getEventCommentText(eventIndex)}: Invalid load net weight (${this.formatCalculationText(weighingValues)}) should be equal to ${loadNetWeight}`,
+      `${this.getEventCommentText(eventIndex)}: Invalid load net weight (${this.formatCalculationText(weighingValues)}) should be approximately ${loadNetWeight}`,
```
Comment (cris-santos) [inline apps/methodologies/bold/rule-processors/mass/net-weight-verification/src/lib/net-weight-verification.processor.ts:74]: To make it clearer, I think we should add the exact difference allowed
